### PR TITLE
Integrate secondary-analysis with to the Grafana Dashboard

### PR DESCRIPTION
**Pull request by rexwangcc**
This PR integrates secondary-analysis (Green Box) to the Grafana dashboard, through adding Terraform definitions.

### CODE REVIEW

File: dashboard.tf [hunk 1]
```diff
@@ -108,6 +108,41 @@ resource "aws_cloudwatch_dashboard" "system_health" {
         "view": "timeSeries",
         "stacked": false
       }
+    },
+    {
+      "type": "metric",
+      "x": 12,
+      "y": 3,
```
Comment (rexwangcc) [inline dashboard.tf]: I'm not sure about the x, y here, it looks different from any of the widgets on the Grafana docs?

File: dashboard.tf [hunk 2]
```diff
@@ -108,6 +108,41 @@ resource "aws_cloudwatch_dashboard" "system_health" {
...
+            "arn:aws:cloudwatch:${var.aws_region}:${data.aws_caller_identity.current.account_id}:alarm:ANALYSIS"
+          ]
+        },
+        "view": "singleValue"
+      }
+    },
+    {
+      "type": "metric",
+      "x": 15,
+      "y": 3,
```
Comment (rexwangcc) [inline dashboard.tf]: Not sure about this either.

**Comment by mweiden** [comment]
@rexwangcc I hadn't merged some of my changes yet. I've doctored up your PR here to fit to my changes. Your health check configuration was correct. I've deployed this to the dev account and will do so for the prod account shortly.

**Comment by rexwangcc** [comment]
@mweiden Thanks for the help and review!

**Pull request merged**